### PR TITLE
Increase Coverage

### PR DESCRIPTION
**Pull request by NotAFile**

### CODE REVIEW

File: tests/piqueserver/test_statistics.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+"""
+test piqueserver/server.py
+"""
+
+from twisted.trial import unittest
+from unittest.mock import Mock
```
Comment (swalladge) [inline tests/piqueserver/test_statistics.py]: `unittest.mock` was introduced in python3.3 - need to use something different or drop python2 support

Comment (NotAFile) [reply tests/piqueserver/test_statistics.py]: that's annoying... I thought I tested it with trials

**Comment by NotAFile** [comment]
Turns out I'm actually just stupid, hm. I commited the tests without commiting the fixes for bugs I wrote while testing them

**Comment by NotAFile** [comment]
the fixes I wrote, not the bugs I wrote

**Pull request merged**